source/Octopus.Tentacle.Tests/Scripts/ExponentialBackoffFixture.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using FluentAssertions;
+using NUnit.Framework;
+using Octopus.Tentacle.Diagnostics;
+using Octopus.Tentacle.Time;
+
+namespace Octopus.Tentacle.Tests.Scripts
+{
+    [TestFixture]
+    public class ExponentialBackoffFixture
+    {
+        readonly Foo foo = new Foo();
```

Comment:
Should this be removed?

Reply:
Don't touch that, its a load bearing `Foo` :stuck_out_tongue:

Reply:
haha... I extracted that to a different class ...